packaging/installer/netdata-uninstaller.sh

```diff
@@ -441,8 +492,29 @@ stop_all_netdata() {
 
 trap quit_msg EXIT
 
+detect_existing_install() {
+  # shellcheck disable=SC2154
+  ndpath="$(PATH="${searchpath}" command -v netdata 2>/dev/null)"
```

Comment:
`searchpath` is not actually defined, so this will usually try searching with an empty `$PATH`, which will always fail.